core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -608,7 +608,7 @@ object AdminUtils extends Logging with AdminUtilities {
   def fetchAllChildEntityConfigs(zkUtils: ZkUtils, rootEntityType: String, childEntityType: String): Map[String, Properties] = {
     def entityPaths(zkUtils: ZkUtils, rootPath: Option[String]): Seq[String] = {
       val root = rootPath match {
-        case Some(path) => rootEntityType + '/' + rootPath
+        case Some(path) => rootEntityType + '/' + path
```

Comment:
Thanks. I don't know, but we can ask Jun. Let's do it in https://github.com/apache/kafka/pull/1987 since it's more focused.